tests/migration/online_migration/pre_migration.pm

```diff
@@ -1,6 +1,6 @@
 # SLE12 online migration tests
 #
-# Copyright 2016 SUSE LLC
+# Copyright 2022 SUSE LLC
```

Comment:
This should be `# Copyright 2016-2022 SUSE LLC` or i believe you can leave intact.